src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.cs

```diff
@@ -1423,15 +1428,15 @@ internal static unsafe IntPtr ComInterfaceForObject(object instance, Guid target
             return comObjectInterface;
         }
 
-        internal static object ComObjectForInterface(IntPtr externalComObject)
+        internal static object ComObjectForInterface(IntPtr externalComObject, CreateObjectFlags flags = CreateObjectFlags.TrackerObject | CreateObjectFlags.Unwrap)
```

Comment:
This is a lot to put in a default parameter. Can't we instead place this where it is needed?